tests/Behat/AuthenticationContext.php

```diff
@@ -147,4 +147,18 @@ public function allowExternalUsers(): void {
     $this->configContext->setConfig('oe_authentication.settings', 'assurance_level', 'LOW');
   }
 
+  /**
+   * Clicks/presses a link or button.
+   *
+   * Drupal 9.3 uses Seven theme but Drupal 9.4 Claro.
```

Comment:
This is unclear, long description of the method should explain the difference like for which theme we need to click/press.